store/server/features/leads/engagement-stage/mutation.ts

```
@@ -0,0 +1,224 @@
+  }
+};
+
+// --- React Query Mutation Hooks ---
+
+/**
+ * Hook for creating an engagement stage.
+ */
+export const useCreateEngagementStage = () => {
+  const queryClient = useQueryClient();
+  return useMutation(createEngagementStage, {
+    onSuccess: (data) => {
```

ESLint

[error] 'data' is defined but never used.

(@typescript-eslint/no-unused-vars)